src/Plugin.php

```diff
@@ -63,6 +63,16 @@ public static function wp_enqueue_scripts() {
     wp_enqueue_script('gallerya-script-custom', static::getBaseUrl() . '/dist/scripts/script.min.js', ['gallerya-script-libs'], FALSE, TRUE);
   }
 
+  public static function post_gallery_init() {
+    $url = home_url(add_query_arg(array()));
+    $id = url_to_postid($url);
+    $post_type = '';
+    if (get_post_type($id) === $post_type || get_post_type() == '') {
+      add_filter('post_gallery', __CLASS__ . '::post_gallery', 10, 2);
+    }
+    return apply_filters('post_gallery_init', $post_type);
```

Comment:
In order to modify the $post_type value I'm using something like:

```
add_filter('post_gallery_init', __CLASS__ . '::post_gallery_init');
public static function post_gallery_init($post_type) {
  $post_type = 'test_value';
  return $post_type;
}
```

But it does not work. 😔  
Where I'm wrong? Can you explain me so next time I can do better?